peripherals/rcc/rcc_f107.yaml

```diff
@@ -0,0 +1,31 @@
+# RCC peripheral
+# Applicable to STM32F107
+
+_include:
+  - ./rcc_common.yaml
+  - ./rcc_pllsrc_1bit.yaml
+
+RCC:
+  CFGR:
+    PLLMUL:
+      Mul4: [2, "PLL input clock x4"]
+      Mul5: [3, "PLL input clock x5"]
+      Mul6: [4, "PLL input clock x6"]
+      Mul7: [5, "PLL input clock x7"]
+      Mul8: [6, "PLL input clock x8"]
+      Mul9: [7, "PLL input clock x9"]
+      Mul6_5: [13, "PLL input clock x6.5"]
+    PLLXTPRE:
+      Div1: [0, "HSE clock not divided"]
+      Div2: [1, "HSE clock divided by 2"]
+    MCO:
+      NoMCO: [0, "MCO output disabled, no clock on MCO"]
+      SYSCLK: [4, "System clock selected"]
+      HSI: [5, "HSI oscillator clock selected"]
+      HSE: [6, "HSE oscillator clock selected"]
+      PLL: [7, "PLL clock selected (divided by 1 or 2, depending en PLLNODIV)"]
```

Comment:
Can we add the remaining MCO options from RM0008, `PLL2`, `PLL3Div2`, `XT1`, `PLL3`?

Reply:
Done.